doc/ref_model/chapters/glossary.md

```diff
@@ -64,8 +64,11 @@ To help guide the reader, this glossary provides an introduction to the terminol
 <a name="11.1.3"></a>
 ### 11.1.3 Operational and Administrative Terminology
 
-- **Tenant**: Tenants represent an independently manageable logical pool of compute, storage and network resources abstracted from physical hardware.
+- **cloud service user**: Natural person, or entity acting on their behalf, associated with a cloud service customer that uses cloud services.
```

Comment:
Can we add "Examples of such entities include devices and applications." to this line, just to call this point out?

Reply:
@tomkivlin Added a note. Thanks